README.md

```diff
@@ -21,20 +21,32 @@ console.log(leblad.getWilayaList());
 
 Returns a list of Algerian provinces (Wilayas)
 
+##### getWilayaByZipCode(zipCode: number, projection?: string[])
+
+Returns a wilaya that includes the given zipCode.
+
 **Arguments**
 
 `projection: string[]` (optional) Array of Wilaya Object attributes.
 
+`zipCode: number` (required) A zip code.
+
```

Comment:
This would be nicer 😄 

```suggestion
**Arguments**

`projection: string[]` (optional) Array of Wilaya Object attributes.

##### getWilayaByZipCode(zipCode: number, projection?: string[])

Returns a wilaya that includes the given zipCode.

**Arguments**

`projection: string[]` (optional) Array of Wilaya Object attributes.
`zipCode: number` (required) A zip code.

```

Reply:
Thank you